po/nl.po

```diff
@@ -0,0 +1,1749 @@
+
+#: data/ui/multiband_compressor.glade:426 data/ui/multiband_gate.glade:450
+msgid "LR4"
+msgstr "LR4"
+
+#: data/ui/multiband_compressor.glade:427 data/ui/multiband_gate.glade:451
+msgid "LR8"
+msgstr "LR8"
+
+#: data/ui/multiband_compressor.glade:441 data/ui/multiband_gate.glade:465
+msgid "Split 1/2"
+msgstr "1/2 splitsen"
+
+#: data/ui/multiband_compressor.glade:454 data/ui/multiband_gate.glade:478
+msgid "Split 2/3"
+msgstr "2/3 splitsen"
+
+#: data/ui/multiband_compressor.glade:467 data/ui/multiband_gate.glade:491
+msgid "Split 3/4"
+msgstr "3/4 splitsen"
+
+#: data/ui/multiband_compressor.glade:854
+#: data/ui/multiband_compressor.glade:1232
+#: data/ui/multiband_compressor.glade:1611
+#: data/ui/multiband_compressor.glade:1990
+msgid "Compression"
+msgstr "Compressie"
+
+#: data/ui/multiband_compressor.glade:937 data/ui/multiband_gate.glade:994
+msgid "Sub Band"
+msgstr "Subband"
+
+#: data/ui/multiband_compressor.glade:1315 data/ui/multiband_gate.glade:1406
+msgid "Low Band"
+msgstr "Lage band"
+
+#: data/ui/multiband_compressor.glade:1694 data/ui/multiband_gate.glade:1819
+msgid "Mid Band"
+msgstr "Middelste band"
+
+#: data/ui/multiband_compressor.glade:2073 data/ui/multiband_gate.glade:2232
+msgid "High Band"
+msgstr "Hoge band"
+
+#: data/ui/multiband_gate.glade:139
+msgid "Multiband Gate"
+msgstr "Multiband-dosering"
+
+#: data/ui/pitch.glade:102
+msgid "Pitch"
+msgstr "Toonhoogte"
+
+#: data/ui/pitch.glade:270
+msgid "Cents"
+msgstr "Cents"
+
+#: data/ui/pitch.glade:300
+msgid "Semitones"
+msgstr "Halve tonen"
+
+#: data/ui/pitch.glade:330
+msgid "Octaves"
+msgstr "Octaven"
+
+#: data/ui/pitch.glade:360
+msgid "Crispness"
+msgstr "Scherpte"
+
+#: data/ui/pitch.glade:402
+msgid "Faster"
+msgstr "Sneller"
+
+#: data/ui/pitch.glade:414
+msgid "Preserve Formant"
+msgstr "Formaat behouden"
+
+#: data/ui/preset_row.glade:55
+msgid "Load"
+msgstr "Laden"
+
+#: data/ui/preset_row.glade:70
+msgid "Save current settings to this preset file"
+msgstr "Sla de huidige instellingen op in dit voorinstellingsbestand"
+
+#: data/ui/preset_row.glade:84
+msgid "Remove this preset file"
+msgstr "Verwijder dit voorinstellingsbestand"
+
+#: data/ui/preset_row.glade:98
+msgid ""
+"Automatically apply this preset whenever the currently used device is "
+"plugged in the system"
+msgstr ""
+"Pas deze voorinstellingen automatisch toe zodra het huidige apparaat wordt "
+"aangekoppeld"
+
+#: data/ui/presets_menu.glade:81 data/ui/presets_menu.glade:182
+#: src/presets_menu_ui.cpp:129
+msgid "Import Presets"
+msgstr "Importeer voorinstellingen"
+
+#: data/ui/pipe_info.glade:77
+msgid "Version"
+msgstr "Versie"
+
+#: data/ui/pipe_info.glade:104
+msgid "Default Sink"
+msgstr "Standaard audiosink"
+
+#: data/ui/pipe_info.glade:116
+msgid "Default Source"
+msgstr "Standaardbron"
+
+#: data/ui/pipe_info.glade:158
+msgid "Protocol"
+msgstr "Protocol"
+
+#: data/ui/pipe_info.glade:185
+msgid "Default Sample Format"
+msgstr "Standaard sampleformaat"
```

Comment:
monsterformaat

Reply:
I see where you're coming from, but in digital music/effects, ‘sample’ is the term that's most commonly used: https://nl.wikipedia.org/wiki/Sample
“Het woord sample komt uit het Engels […]  Als zodanig wordt de term gebruikt in de statistiek, bij digitale signaalbemonstering en in de context van digitale muziekinstrumenten. In sommige vakgebieden wordt de term gebruikt voor fragmenten of testresultaten die een groter geheel omvatten, bijvoorbeeld door musici en ecologen.”